### ISSUE TITLE
Hangs at startup on Windows + work around

### ISSUE THREAD

**Issue by quintesse** [sleuthkit/autopsy#7918]
Not really sure what's going on but after installing and running the app there's this dialog that pops up that's completely white (no borders, no icons, no titlebar) and that's inactive (tried clicking everywhere).

Waited for a long time but it wouldn't continue so the only thing I could do was to kill the process.

I then tried running it from a console to see if there were any errors, but none showed, just a bunch of very normal looking log messages.

Finally, seeing there was also a bash script in the same folder as the autopsy64.exe I fired up my Git Bash and started the app using the script and everything worked fine.

**Comment by Geozstevenzz** [comment]
Same problem. Application dowsnt launch and stays suspended in task manager. 
![Screenshot_8](https://github.com/user-attachments/assets/2d19a30a-9070-4e15-b042-653d8bdef1ad)
Dead white screen.

**Comment by simsong** [comment]
How are you making the application? Are you compiling from sources or downloading the release? Do you see this behavior on the math machine? Do you see this behavior with the official distribution on your machine? How about with the official distribution on another machine?

**Comment by markmckinnon** [comment]
Can you move the white screen around to see if there is a pop up behind it that needs attention.

**Comment by quintesse** [comment]
@markmckinnon no, there's nothing behind it.

Edit: also, you can't right-click and select close on the taskbar icon either, it will just ignore it. You have to manually kill it using the task manager.

**Comment by markmckinnon** [comment]
Can you post the messages.log file from the following
directory C:\Users\<User Name>\AppData\Roaming\autopsy\var\log

**Comment by quintesse** [comment]
Sure, here you are (lots of exceptions!)

[messages.log](https://github.com/user-attachments/files/18157610/messages.log)

**Comment by Geozstevenzz** [comment]
Here is mine as well
[messages.log](https://github.com/user-attachments/files/18161406/messages.log)

There is no pending dialogue box behind the white screen and I am using the release version. I am not building

**Comment by markmckinnon** [comment]
@Geozstevenzz @quintesse  Do either one of you have JDK or JRE installed as a standalone app?  If so what versions?

**Comment by Geozstevenzz** [comment]
Version 17.0.10
![Uploading Screenshot_10.png…]()

**Comment by markmckinnon** [comment]
@Geozstevenzz is that installed to fullfill a requirement for some other software on your computer.  if not then what happens if you uninstall it.  If it is a requirement for some other software then do not uninstall it.  Autopsy comes packaged with a JRE so it does not require you to install java to run it.

**Comment by quintesse** [comment]
@markmckinnon yeah, I'm pretty sure our local JDKs are getting preference over the included JDK. Because my default JDK is 17.0.11, if I switch it to 11.0.23 Autopsy starts correctly (which I'm guessing works because in 11 JavaFX was still distributed with the JDK by default while from 17 onwards that is no longer the case)

**Comment by markmckinnon** [comment]
@quintesse I would agree that is what is happening in your case.  Is your local JDKs also included in your PATH?  If you could remove that temporarily then you can see if Autopsy would start then.

**Comment by quintesse** [comment]
@markmckinnon yes, in that case it starts just fine.
